### ISSUE TITLE
Add a note for the limitation of authentication

### ISSUE THREAD

**Issue by hihayak** [MicrosoftDocs/msix-docs#215]
[Enter feedback here]
Please consider to add a note for the limitation of authentication if exists. Enterprise developers wonder whether MSIX supports proxy authentication. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fd3a8483-a56e-7f3d-8c25-1d8b18943342
* Version Independent ID: 3b8e451b-95b5-3d26-c7ec-160fef01d85e
* Content: [Troubleshoot installation issues with the App Installer file - MSIX](https://review.docs.microsoft.com/en-us/windows/msix/app-installer/troubleshoot-appinstaller-issues?branch=main)
* Content Source: [msix-src/app-installer/troubleshoot-appinstaller-issues.md](https://github.com/microsoftdocs/msix-docs/blob/master/msix-src/app-installer/troubleshoot-appinstaller-issues.md)
* Product: **msix**
* GitHub Login: @dianmsft
* Microsoft Alias: **diahar**

**Comment by tcrevier** [comment]
Agreed, if you follow the example (https://docs.microsoft.com/en-us/windows/msix/app-installer/web-install-azure) for using an Azure web site to host an installer, it doesn't work if you have Azure AD authentication.  Not really clear how one would use this if you cannot use authentication...
